src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.c

```diff
@@ -13,17 +13,16 @@
 void BgGndDarkmeiro_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgGndDarkmeiro_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgGndDarkmeiro_Update(Actor* thisx, GlobalContext* globalCtx);
+void BgGndDarkmeiro_Draw0(Actor* thisx, GlobalContext* globalCtx);
+void BgGndDarkmeiro_DrawSwitchBlock(Actor* thisx, GlobalContext* globalCtx);
+void BgGndDarkmeiro_DrawStaticBlock(Actor* thisx, GlobalContext* globalCtx);
 
-void func_80878C30(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_80878F28(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_80878F34(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_80879128(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_80879134(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_808791BC(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_808791E8(BgGndDarkmeiro* this, GlobalContext* globalCtx);
-void func_80879334(BgGndDarkmeiro* this, GlobalContext* globalCtx);
+void BgGndDarkmeiro_ToggleBlock(BgGndDarkmeiro* this, GlobalContext* globalCtx);
```

Comment:
This prototype can be omitted, the function's body is declared before being used

Reply:
I prefer to just have all prototypes at the top, rather than determine which ones are needed.

Reply:
Eh, scratch that